Framework/Source/API/D3D/D3D12/D3D12Texture.cpp

```diff
@@ -106,7 +106,18 @@ namespace Falcor
         desc.Width = align_to(getFormatWidthCompressionRatio(mFormat), mWidth);
         desc.Height = align_to(getFormatHeightCompressionRatio(mFormat), mHeight);
         desc.Flags = getD3D12ResourceFlags(mBindFlags);
-        desc.DepthOrArraySize = (mType == Texture::Type::TextureCube) ? mArraySize * 6 : mArraySize;
+        if (mType == Texture::Type::TextureCube)
```

Comment:
Since this is no longer a one-liner, can you move this a few lines lower so it comes after the block of `desc. ... = ...;` code?